mail_activity_reminder/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Mail Activity Reminder',
+    'version': '12.0.1.0.0',
+    'category': 'Discuss',
+    'website': 'https://github.com/OCA/social',
+    'author':
+        'Brainbean Apps, '
+        'Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'installable': True,
+    'application': False,
+    'summary': 'Reminder notifications about planned activities',
+    'depends': [
+        'mail',
+    ],
+    'external_dependencies': {
```

Comment:
This should not stay here as is not a functional dependency (I wish we have test only dependencies here :/ ).
If you want to not include this lib in your prod build you should be able to. This prevents it.
MQT=PIP + test-requirerements.txt allows it.

Reply:
> Now MQT recently gained a new way to discover dependencies automatically, and incidentally has a mechanism to declare test dependencies at the repo level. (note this will ignore requirements.txt and oca_dependencies.txt)

> So in this case you can add freezegun to test-requirements.txt and set MQT_DEP=PIP in .travis.yml.

Did I get it right that having `test-requirerements.txt` will completely ignore `requirements.txt` and `oca_dependencies.txt` and leading to quite an unpleasant things?

Reply:
if your dependencies are properly declared, you'll get the same result because you'll have functional dependencies installed via PIP and test deps installed via requirements.
In any case, even if you don't want to use MQT=PIP here, you should remove the dependency.

Reply:
What I want is to get over this module finally :D

Reply:
Added, now it's not working. What did I do wrong?

Reply:
Oh, misspelled the file name

Reply:
Done! Can we please merge it now? :)